docs/_build/triggers/deduplication.md

```diff
@@ -111,10 +113,11 @@ return items.map((item) => {
 {% endraw %}
 {% endhighlight %}
 
+Alternatively, if you're using the CLI, you can use non-id fields as the primary key. See ["How does deduplication work?"](https://github.com/zapier/zapier-platform/blob/main/packages/cli/README.md#how-does-deduplication-work) in the CLI docs for example code.
 
-## Updated item triggers
+## Updated Item triggers
 
-When triggering on updated items, you'll want to define the `id` field to be used for deduplication. Assuming your task API has an endpoint that can return tasks sorted by `updatedAt` in descending direction, here's example code to incorporate the `updatedAt` value with the `id`, so that Zapier recognizes a new update as a new item. Assuming that you have configured `options` with the appropriate API request URL and parameters:
+When triggering on updated items, you'll want to define the `id` field or a primary key to be used for deduplication. Assuming your task API has an endpoint that can return tasks sorted by `updatedAt` in descending direction, here's example code to incorporate the `updatedAt` value with the `id`, so that Zapier recognizes a new update as a new item. Assuming that you have configured `options` with the appropriate API request URL and parameters:
```

Comment:
Perhaps here add a note how in CLI you'd set `primary: true` for both the original `id` and `updatedAt`, instead of overriding the value of `id`.

Reply:
Sounds good. Updated in [55c2528](https://github.com/zapier/visual-builder/pull/596/commits/55c25288e2a3440def9c051a3ca41b737e585cfe).